tests/unit/makefile

```diff
@@ -24,7 +24,8 @@ CCC = $(CC) $(CCOPTS)
 # -I$(MFEM_DIR) is needed by some tests, e.g. to #include "general/text.hpp"
```

Comment:
Does the CMake build system also need to be updated?

Reply:
OMG, I hope not, I would need help on that...